addons/l10n_id/models/qris_transaction.py

```diff
@@ -0,0 +1,118 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import pytz
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+# Using different field for reference for different modelsa.
+# IN PoS live session, no records are created in the backend until payment is completed
+# and therefore we can'use id for pos.order. However PoS generates unique uuid for pos.order which
+# later on will be included when creating pos.order, hence the variable.
+MODEL_ID_FIELDS = {
+    'account.move': 'id',
+    'pos.order': 'uuid'
+}
```

Comment:
Please make a method to get that, or just use _get_record

And override it in the pos module to check the model and get the record correctly based on that.